der/src/decode.rs

```diff
@@ -212,14 +212,20 @@ where
 }
 
 #[cfg(feature = "alloc")]
-impl<'a, T> DecodeValue<'a> for Cow<'a, T>
+impl<'a, T, E> DecodeValue<'a> for Cow<'a, T>
```

Comment:
This still somewhat annoyingly ties the lifetime of the `Cow` to the lifetime of the `Reader`, which for `PemReader` is still the lifetime of the input buffer.

Reply:
Aha, that's easily fixable by making it impl `Reader<'static>` instead